lib/eco/export/jruby.py

```diff
@@ -241,19 +241,21 @@ def _profile(self):
                                      next(tempfile._get_candidate_names()) + ".txt")
         logging.debug("Placing callgraph trace in", log_file_name)
 
-        # Run this command:
-        #  $ jruby -X+T -Xtruffle.callgraph=true -Xtruffle.callgraph.write=test.txt -Xtruffle.dispatch.cache=2 FILE
         settings = QSettings("softdev", "Eco")
         jruby_bin = str(settings.value("env_jruby", "").toString())
+        directory = str(settings.value("env_jruby_load", "").toString())
+        load_path = "-I" + directory if directory else ""
```

Comment:
I find Python's ternary operator a bug-inducing disaster. Can we make it a standard if/then/else? Thanks!